openlibrary/templates/search/lists.html

```diff
@@ -21,6 +21,6 @@ <h1>$_("Lists")</h1>
       </ul>
     </div>
   $elif q:
-    <div>No results, <a href="/search/inside?q=$q">try a full-text search?</a></div>
+    <div>$:_('No results, <a href="/search/inside?q=$q">try a full-text search?</a>')</div>
```

Comment:
Ah this one's a bit annoying because `q` is user input, and `$:` + user input introduces a risk of someone injecting. Actually, let's switch this to behave like the normal search page:

```suggestion
    <div>
      <span class="red">$_("No results found.")</span>
      <a href="/search/inside?$urlencode(dict(q=q))">$_('Search for books containing the phrase "%s"?', q)</a>
    </div>
```

(oops! Noticed that page has a bug in this string, but this should be correct.